src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix3x2.cs

```diff
@@ -111,19 +123,20 @@ public Vector2 Translation
                  && value1.M32 == value2.M32);
         }
 
-        /// <summary>Returns a boolean indicating whether the given matrices are not equal.</summary>
-        /// <param name="value1">The first source matrix.</param>
-        /// <param name="value2">The second source matrix.</param>
-        /// <returns>True if the matrices are not equal; False if they are equal.</returns>
+        /// <summary>Returns a value that indicates whether the specified matrices are not equal.</summary>
+        /// <param name="value1">The first matrix to compare.</param>
+        /// <param name="value2">The second matrix to compare.</param>
+        /// <returns><see langword="true" /> if <paramref name="value1" /> and <paramref name="value2" /> are not equal; otherwise, <see langword="false" />.</returns>
         public static bool operator !=(Matrix3x2 value1, Matrix3x2 value2)
         {
             return !(value1 == value2);
         }
 
-        /// <summary>Multiplies two matrices together and returns the resulting matrix.</summary>
-        /// <param name="value1">The first source matrix.</param>
-        /// <param name="value2">The second source matrix.</param>
+        /// <summary>Returns the matrix that results from multiplying two matrices together.</summary>
```

Comment:
I don't think we should block, but I'm really not a fane of this style of summary. I think `Multiples` first, as the action the method is perform, reads significantly better

Reply:
@tannergooding if you can give me comment suggestions with the texts you prefer to see, that would be great. That way I can batch apply them.

Reply:
I think either `Multiplies two matrices together to compute the product` is simple, explanatory, and puts it in common math terms.

Likewise, `Divides two matrices to compute the quotient`, `Adds two matrices to compute the sum`, `Subtracts two matrices to compute the difference`.

And when the types differ: `Multiplies a matrix by a float to compute the product` or `Divides a matrix by a float to compute the quotient`, etc

Reply:
(Assuming, of course, the docs team doesn't see any issues here)

Reply:
Tanner's suggestions sound great to me.

Reply:
I addressed the ones I could find in Matrix3x2 and in Matrix4x4. If you find more, please help me by using the `` ```suggestion`` feature from GitHub.